test/http_client_test.exs

```diff
@@ -34,6 +34,11 @@ defmodule HTTPClientTest do
       assert resp.request_url == "http://localhost:8082/foo%20bar?baz=%20true"
     end
 
+    test "the request headers contain a User-Agent header with the value MozartFetcher" do
+      {:ok, resp} = HTTPClient.get("http://localhost:8082/foo/bar")
+      assert Enum.member?(resp.request.headers, {'User-Agent', 'MozartFetcher'}) == true
```

Comment:
The following should also be possible, just to beef up your future options : )

```ex
  assert  {'User-Agent', 'MozartFetcher'} in resp.request.headers
```

Reply:
Also, the following is idiomatic (not needing`== true`) as `Enum.member?/2` is a boolean function

```ex
  assert Enum.member?(resp.request.headers, {'User-Agent', 'MozartFetcher'})
```